src/catch2/catch_config.hpp

```diff
@@ -20,6 +22,25 @@ namespace Catch {
     struct IStream;
 
     struct ConfigData {
+        struct ReporterAndFile {
+            std::string reporterName;
+
+            // If none, the output goes to the default output.
+            Option<std::string> outputFileName;
+
+            friend bool operator==(ReporterAndFile const& a, ReporterAndFile const& b) {
```

Comment:
Nit: I strongly prefer `lhs` and `rhs` 😃

Reply:
Done.